### PR TITLE
fix: Fix ScrollBarVisibility regression on Wasm

### LINKED ISSUE #12889: [WASM] FlipView scroll bars are visible
### Current behavior

image

In the latest package versions, the FlipView shows horizontal/vertical scroll bars.

### Expected behavior

Scroll bars are not visible.

### How to reproduce it (as minimally and precisely as possible)

http://playground.platform.uno/#553281e6
[SampleApp.zip](https://github.com/unoplatform/uno/files/12044121/SampleApp.zip)

### Workaround

_No response_

### Works on UWP/WinUI

Yes

### Environment

_No response_

### NuGet package version(s)

"Uno.Resizetizer" Version="1.2.0-dev.4"
"Microsoft.Windows.Compatibility" Version="7.0.0"
"Uno.Toolkit.WinUI" Version="4.0.0-dev.49"
"Uno.Toolkit.WinUI.Material" Version="4.0.0-dev.49"
"Uno.Wasm.Bootstrap" Version="8.0.0-dev.228"
"Uno.Wasm.Bootstrap.DevServer" Version="8.0.0-dev.228"
"Uno.WinUI.WebAssembly" Version="5.0.0-dev.1392"
"Uno.WinUI.RemoteControl" Version="5.0.0-dev.1392" Condition="'$(Configuration)'=='Debug'"
"Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="5.0.0-dev.1392"
"CommunityToolkit.Mvvm" Version="8.1.0"
"Uno.Extensions.Logging.WebAssembly.Console" Version="1.6.0-dev.2"
"Microsoft.Extensions.Logging.Console" Version="7.0.0"

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

### IDE version

17.6.3

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #12889

## PR Type

What kind of change does this PR introduce?

### <samp>🤖 Generated by Copilot at 992d22c</samp>

Fixed a bug in WASM that caused incorrect visibility of scroll bars in `ScrollContentPresenter`. Updated the CSS class logic to match the `ScrollBarVisibility` enum values.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs [hunk 1]
```diff
@@ -112,7 +118,7 @@ private void HandlePointerEventIfOverNativeScrollbars(Input.PointerRoutedEventAr
 			}
 		}
 
-		private static readonly string[] VerticalVisibilityClasses = { "scroll-y-auto", "scroll-y-disabled", "scroll-y-hidden", "scroll-y-visible" };
+		private static readonly string[] VerticalVisibilityClasses = { "scroll-y-disabled", "scroll-y-auto", "scroll-y-hidden", "scroll-y-visible" };
```
Comment (ajpinedam) [inline src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs:121]: Good finding!

**Review by ajpinedam — APPROVED** [review]
🎉

**Review by MartinZikmund — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@Mergifyio refresh

**Comment by Youssef1313** [comment]
Closing and re-opening to get license/cla check unblocked.

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**